src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Disabled Context auto saving when token cache persistence fails
```

Comment:
How about make it more precise by adding "on Windows and MacOS"

Reply:
Done